internal/mempool/mempool.go

```diff
@@ -278,8 +278,11 @@ func (txmp *TxMempool) CheckTx(
 	}
 
 	txmp.defaultTxCallback(tx, res)
-	txmp.initTxCallback(wtx, res, txInfo)
+	err = txmp.initTxCallback(wtx, res, txInfo)
```

Comment:
Do you mind creating an issue after this is merged reminding us to clean up the callbacks we have in the mempool for 0.37